siFi: Software for long double-stranded RNAi-target design and off-target prediction

Version v1.2.3-0008

Homepage: http://www.snowformatics.com/si-fi.html

Stefanie LÃ¼ck, Tino Kreszies, Marc Strickert, Patrick Schweizer, and Dimitar Douchkov

RNA interference (RNAi) is a nucleic acid complementarity-based biological phenomenon and a widespread natural mechanism for the inhibition or regulation of gene expression. RNAi is an essential part of the immune response to viruses and other foreign genetic material especially in plants but also in many fungal and animal species where it is a part of their innate immunity system. RNAi has become an important research tool for studying gene function by strong and selective suppression of the genes of interest also in large-scale screens. However, the application of RNAi as a technology raises important questions about efficiency and specificity of corresponding gene constructs. Since the RNAi machinery selects targets based on sequence similarity, there is an inherent risk of hitting non-targeted genes. Therefore a reliable search and prediction of off-targets is crucial for the practical application of RNAi. Besides being gene-specific, a successful RNAi construct should also be able to produce a strong silencing effect, which is highly depending on selecting an optimal sequence for design of the RNAi construct.
Here, we report on a software called siFi for optimizing long double-stranded RNAi- target design and for prediction of RNAi off-targets. It is open source desktop software that provides an intuitive graphical user interface, works in Microsoft Windows environment and can use custom sequence databases in  standard FASTA format. 

Attribution-NonCommercial-ShareAlike 2.0 Generic (CC BY-NC-SA 2.0) License
